exporters/tests/test_failure_exporter.py

```diff
@@ -189,6 +189,7 @@ def test_jira_search_with_wrong_jql():
     assert len(issues) == 0
 
 
+@pytest.mark.skip(reason="Issue-848")
```

Comment:
remove this skip, as it was fixed

Reply:
Removed. Will push in few once tests passes.